### PR TITLE
Add reuse compliance

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* SPDX header
* GH action
* Readme badge
* Updated issue templates
* Updates icons for Material Symbol variant
* checked that files from 2015/2016 have an oC copyright line if created before June 2016

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.md
Comment (nickvergessen) [inline .github/ISSUE_TEMPLATE/1_bug_report.md:1]: If we use the default, we don't need to copy it

Comment (AndyScherzinger) [reply .github/ISSUE_TEMPLATE/1_bug_report.md:1]: removed via https://github.com/nextcloud/announcementcenter/pull/861/commits/1f864dbceece3f7cb4bc4cf16f5cd9e3326463a6

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: 2015-2026 ownCloud, Inc.
```
Comment (nickvergessen) [inline .scrutinizer.yml:1]: wrong, but also unused, will remove

**Review by nickvergessen — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
Sweet 🎉
I wasn't aware that codeowner files allow for comments 🎉

**Pull request merged**